jsapi/README.md

```diff
@@ -1,16 +1,29 @@
 # cardano-addresses TypeScript binding
 
+- 📦 [NPM Package](https://www.npmjs.com/package/cardano-addresses)
+- 📗 [API Documentation](https://input-output-hk.github.io/cardano-addresses/typescript/)
+- 🎇 [Web demo](https://input-output-hk.github.io/cardano-addresses/demo/)
+- λ [Haskell Library](https://input-output-hk.github.io/cardano-addresses/haddock/)
+
 This is a Typescript (or Javascript) version of the
-`caardano-addresses` API. It comprises of three parts:
+[`cardano-addresses`](https://github.com/input-output-hk/cardano-addresses)
+API. It comprises of three parts:
 
  * The ghcjs build of `cardano-addresses:library` (supported by the
    `cardano-addresses-jsbits` package.
- * A Cabal package `cardano-addresses-jsapi` containing GHCJS
+
+* A Cabal package `cardano-addresses-jsapi` containing GHCJS
    foreign exports for translating Javascript function calls and
    values into Haskell function calls and values, and vice-versa.
- * An NPM package `cardano-addresses` containing NodeJS modules written in TypeScript, which thinly wraps the GHCJS foreign exports to make a proper API.
 
-## Quick Start
+* An NPM package `cardano-addresses` containing NodeJS modules
```

Comment:
```suggestion
* An NPM package `cardano-addresses` containing ES modules
```

Reply:
OK fixed. Actually there are both CommonJS and ES modules.